src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -702,8 +748,8 @@ public void UpdatePackageStatus(IEnumerable<PackageCollectionItem> installedPack
                 .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageVersionsAsync));
 
             NuGetUIThreadHelper.JoinableTaskFactory
-                .RunAsync(ReloadPackageDeprecationAsync)
-                .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageDeprecationAsync));
+                .RunAsync(ReloadPackageMetadataAsync)
+                .PostOnFailure(nameof(PackageItemViewModel), nameof(ReloadPackageMetadataAsync));
```

Comment:
I see that you have replaced a call to `ReloadPackageDeprecationAsync` with `ReloadPackageMetadataAsync` methods. Now `ReloadPackageDeprecationAsync` method have no callers. Hence the method can be removed I think.

Reply:
Good catch, thanks! I meant to clean that up since there's no longer any references to that code.